convert2rhel/main.py

```diff
@@ -201,6 +201,12 @@ def pre_ponr_conversion():
         loggerinst.task("Convert: Install RHEL certificates for RHSM")
         system_cert = cert.SystemCert()
         system_cert.install()
+
+    # remove non-RHEL packages containing repofiles or affecting variables in the repofiles
+    loggerinst.task("Convert: Remove packages containing .repo files")
+    pkghandler.remove_repofile_pkgs()
+
+    if not toolopts.tool_opts.no_rhsm:
```

Comment:
Add a comment about needing to remove the repofile packages before disabling repos